larray/core.py

```diff
@@ -9358,44 +9358,44 @@ def read_csv(filepath, nb_index=None, index_col=None, sep=',', headersep=None, n
     return df_aslarray(df, sort_rows=sort_rows, sort_columns=sort_columns, fill_value=na, raw=raw)
 
 
-def read_tsv(filepath, **kwargs):
-    return read_csv(filepath, sep='\t', **kwargs)
+def read_tsv(filepath_or_buffer, **kwargs):
+    return read_csv(filepath_or_buffer, sep='\t', **kwargs)
 
 
-def read_eurostat(filepath, **kwargs):
+def read_eurostat(filepath_or_buffer, **kwargs):
     """Reads EUROSTAT TSV (tab-separated) file into an array.
 
     EUROSTAT TSV files are special because they use tabs as data separators but comas to separate headers.
 
     Parameters
     ----------
-    filepath : str
-        Path to the file.
+    filepath_or_buffer : str or any object with a read() method
+        Path where the csv file has to be read or a file handle or a StringIO object.
     kwargs
         Arbitrary keyword arguments are passed through to read_csv.
 
     Returns
     -------
     LArray
     """
-    return read_csv(filepath, sep='\t', headersep=',', **kwargs)
+    return read_csv(filepath_or_buffer, sep='\t', headersep=',', **kwargs)
 
 
-def read_hdf(filepath, key, na=np.nan, sort_rows=False, sort_columns=False, **kwargs):
+def read_hdf(path_or_buf, key, na=np.nan, sort_rows=False, sort_columns=False, **kwargs):
     """Reads an array named key from a HDF5 file in filepath (path+name)
 
     Parameters
     ----------
-    filepath : str
-        Filepath and name where the HDF5 file is stored.
+    path_or_buf : str or buffer
```

Comment:
the argument name should be coherent. Either filepath_or_buffer everywhere or path_or_buf everywhere (or path_or_buffer). I do not really care either way.